package.json

```
@@ -3,6 +3,8 @@
   "scripts": {
     "build": "next build",
     "dev": "next dev",
+    "lint": "next lint",
+    "lint:fix": "next lint --fix",
     "start": "next start"
   },
```

```suggestion
    "start": "next start"
    "prepare": "husky install"
  },
```
Do we need a "prepare" script to enable the Git hooks? Pre-commit didn't run for me otherwise.